test/playback.test.js

```diff
@@ -0,0 +1,74 @@
+import QUnit from 'qunit';
+import videojs from 'video.js';
+/* eslint-disable no-unused-vars */
+import { Hls } from '../src/videojs-contrib-hls';
+
+let when = function(element, type, cb, condition) {
+  element.on(type, function func() {
```

Comment:
I want the function to fire when the targeted event is triggered and a certain condition is met. Using `one` will just remove it the first time the event is fired.

For example, I use this with `timeupdate` and some target time.